docs/dabp/release-notes/index.md

```diff
@@ -0,0 +1,11 @@
+# Release notes
+
+This section contains release notes for Onegini DABP.
+
+In the release notes we mention new features and bug fixes. If anything is unclear, feel free to contact [Onegini Support](https://support.onegini.com).
+
+## Release date: 2021-04-13
+* Introduced log events via oneex
+* After removing a user from group, if he is no longer of any group his account will be removed from CIM
+* Translation fixes and improvements
+* Minor UI improvements
```

Comment:
** Color of the Profile icon is now configurable
** Logo format has changed from .png to .svg
** A preview is shown when the user details are loaded
** User status and app loading messages are now translatable
** Forms don't close anymore after inactivity
** Error message on "Add user" page will clear after from is closed
** Attributes are sorted by name
** More resilience against error 504

Reply:
added